metaflow/datatools/s3.py

```diff
@@ -29,7 +29,7 @@
     # python3
     from urllib.parse import urlparse
 
-from metaflow.datastore.util.s3util import get_s3_client
+from .s3util import get_s3_client
```

Comment:
can we get rid of `datastore.util` now?

Reply:
Yes (sorry, forgot to erase the __init__.py file but yes, it is no longer needed. I moved it to datatools.